### PR TITLE
[LTC] Implement Summons of Saruman

### PR DESCRIPTION
**Pull request by Susucre**

### CODE REVIEW

**Comment by Susucre** [comment]
[[Summons of Saruman]]

File: Mage.Sets/src/mage/cards/s/SummonsOfSaruman.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+/**
+ * @author Susucr
+ */
+public final class SummonsOfSaruman extends CardImpl {
+
+    public SummonsOfSaruman(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{X}{U}{R}");
+
+        // Amass Orcs X.
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/s/SummonsOfSaruman.java:34]: Kinda weird the comment doesn't have the full rules text

File: Mage.Sets/src/mage/cards/i/InvadeTheCity.java [hunk 2]
```diff
@@ -49,7 +49,7 @@ class InvadeTheCityEffect extends OneShotEffect {
     InvadeTheCityEffect() {
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/i/InvadeTheCity.java:49]: Can this be the common Amass effect with `CardsInControllerGraveyardCount(StaticFilters.FILTER_CARD_INSTANT_AND_SORCERY)`? Probably still needs text override. (There are probably other cards that could be simplified in a similar way in the future.)

File: Mage.Sets/src/mage/cards/s/SummonsOfSaruman.java (hunk 1 above)
Comment (Susucre) [reply Mage.Sets/src/mage/cards/s/SummonsOfSaruman.java:34]: I'll group the 2 comments, as they are 1 line on the card.

File: Mage.Sets/src/mage/cards/i/InvadeTheCity.java (hunk 2 above)
Comment (Susucre) [reply Mage.Sets/src/mage/cards/i/InvadeTheCity.java:49]: Oh I did miss that one when searching for amass X.
Most of the others are amassing inside a `OneShotEffect`, along with other code and I didn't want to break any card.

Comment (Susucre) [reply Mage.Sets/src/mage/cards/i/InvadeTheCity.java:49]: Oh there was Barad-Dur too. Now that I understand the power of `ManacostVariableValue.REGULAR`

**Pull request merged**